packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -261,7 +261,7 @@ export interface IContextualMenuItem {
    * Method to custom render this menu item
    * @defaultvalue undefined
    */
-  onRender?: (item: any) => React.ReactNode;
+  onRender?: (item: any, index?: number, hasCheckmarks?: boolean, hasIcons?: boolean) => React.ReactNode;
```

Comment:
Can we type `item`? And add jsdoc comments for `hasCheckmarks` and `hasIcons`? might be confusing since these are not item scoped but apply to the context menu in general